### PR TITLE
Help install snap

### PR DESCRIPTION
**Pull request by earnubs**
Help 'install snap' component.

### CODE REVIEW

File: src/common/components/help/install-snap/index.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import React, { Component, PropTypes } from 'react';
+
+import { HeadingSix } from '../../vanilla/heading/';
```
Comment (earnubs) [inline src/common/components/help/install-snap/index.js]: This could be a little less declarative?

File: src/common/containers/build-details.js [hunk 2]
```diff
@@ -60,6 +61,7 @@ class BuildDetails extends Component {
             <BuildLog logUrl={build.buildLogUrl} />
           </div>
         }
+        <HelpInstallSnap name='foo' revision='1' />
```
Comment (earnubs) [inline src/common/containers/build-details.js]: Stubbed out for now, blocked on need to get snap name and revision into redux.

File: src/common/style/base.css [hunk 3]
```diff
@@ -49,8 +49,8 @@ q:before, q:after {
 }
 
 a {
-  color: #dd4814;
-  text-decoration: none;
+  color: $cool-grey;
```
Comment (earnubs) [inline src/common/style/base.css]: Global change, but seems to be what the layouts have.

File: src/common/style/base.css [hunk 4]
```diff
@@ -64,47 +64,3 @@ a:focus {
 body {
   background-color: #FFF;
 }
-
-h1,
```
Comment (earnubs) [inline src/common/style/base.css]: These just get in the way of we're using react components for h1-6, also they're not in line with current Vanilla afaics.

File: src/common/components/help/install-snap/index.js (hunk 1 above)
Comment (katiefenn) [inline src/common/components/help/install-snap/index.js]: They could be. When they're as small and as simple as this, the difference seems marginal.

Comment (earnubs) [inline src/common/components/help/install-snap/index.js]: Yeah, but there's a lot of room for error in all this boilerplate.

File: src/common/style/base.css (hunk 3 above)
Comment (bartaz) [inline src/common/style/base.css]: @earnubs Layout are grayscale wireframes and Vanilla has coloured links, so I'm not so sure about that...

**Pull request merged**